.github/workflows/integration-tests.yml

```diff
@@ -5,7 +5,8 @@ jobs:
   changes:
     runs-on: ubuntu-latest
     outputs:
-      src: ${{ steps.changes.outputs.src }}
+      # src: ${{ steps.changes.outputs.src }}
+      src: 'true'
```

Comment:
Should this be reverted?

Reply:
Ya, this will get reverted, just need to ensure that the tests are getting run to see if things are working with new names for now.